geode-core/src/distributedTest/java/org/apache/geode/internal/cache/execute/FunctionRetryTestBase.java

```diff
@@ -262,11 +262,6 @@ private Execution getExecutionTarget(ExecutionTarget executionTarget) {
         execution = FunctionService.onServers(clusterStartupRule.getClientCache().getDefaultPool())
             .setArguments(200);
         break;
-      case SERVERS_REGION_SERVICE:
```

Comment:
Why is this case statement is removed?

Reply:
@agingade It was a duplicate as far as we could tell.

Reply:
@agingade but on second look I see it is `SERVER` vs. `SERVERS`. I'll push a fix.

Reply:
Also it should not be referring to GEODE-6677. If there is a need to backport or analyze the changes for GEODE-6588; this should be part of that.